tests/test_z3.py

```diff
@@ -153,3 +159,18 @@ def test_case() -> None:
             z3.If(z3.Int("x") == z3.IntVal(10), z3.IntVal(2), z3.BoolVal(False)),
         ),
     )
+
+
+def test_attribute() -> None:
+    assert (Size("X")).z3expr() == z3.Int("X'Size")
+    assert (-Size("X")).z3expr() == -z3.Int("X'Size")
+    assert (Length("X")).z3expr() == z3.Int("X'Length")
+    assert (-Length("X")).z3expr() == -z3.Int("X'Length")
+    assert (First("X")).z3expr() == z3.Int("X'First")
+    assert (-First("X")).z3expr() == -z3.Int("X'First")
+    assert (Last("X")).z3expr() == z3.Int("X'Last")
+    assert (-Last("X")).z3expr() == -z3.Int("X'Last")
+    assert (Range("X")).z3expr() == z3.Int("X'Range")
+    assert (-Range("X")).z3expr() == -z3.Int("X'Range")
+    assert (Old("X")).z3expr() == z3.Int("X'Old")
+    assert (-Old("X")).z3expr() == -z3.Int("X'Old")
```

Comment:
Nit: This test could be simplified by using `@pytest.mark.parametrize`.

Reply:
Good idea. I'm not used to that mechanism, yet, even though it's very powerful. Changed in f48e9d6.